lib/src/components/NxTag/NxTag.scss

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
+ @import '../../scss-shared/nx-container-helpers';
+
+.nx-tag {
+  @include semi-bold();
+  align-items: center;
+  background-color: $nx-blue-800;
+  border: 1px solid $nx-blue-800;
+  border-radius: 16px;
```

Comment:
I think we need an explicit `height` to go along with this to ensure that the border-radii match up to make proper semi-circles.  In particular the tags are currently 34px tall which doesn't seem right.  I suspect the padding numbers from zeplin are not accounting for the 1px border.

Reply:
> I think we need an explicit `height` to go along with this to ensure that the border-radii match up to make proper semi-circles. In particular the tags are currently 34px tall which doesn't seem right. I suspect the padding numbers from zeplin are not accounting for the 1px border.

https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/c13ff544cb89da4ad79a33f6d4e5f8083afb193e

Reply:
> c13ff54

With that, the font is no longer vertically centered.  I suggest removing the top and bottom padding and increasing the line-height to 30px (the full height of the tag minus borders).

Reply:
Zeplin is showing that the tags are 32px tall (which is what they should be) and only the deselected state for the color tags have borders. If you're still running into issues then I'm fine with going with @rpokorny suggestion

Reply:
> Zeplin is showing that the tags are 32px tall (which is what they should be) and only the deselected state for the color tags have borders. If you're still running into issues then I'm fine with going with @rpokorny suggestion

Just FYI for @dturner32 this is an implementation detail but *all* tags have borders, it's just that the unelectable, and selected tags' borders are the same colour as the background. We do this so we don't have a situation where the size of the tag changes when the borders are added or removed or so that we don't have to do fancy things like adjusting the tag height to compensate for the presence of borders. It would be more apt to say that only the unselected tags have _visible_ borders.